### PR TITLE
Rm 9 enable test 3.7 job in circle ci config fix typecheck problems

### PR DESCRIPTION
**Pull request by ryantimjohn**

### CODE REVIEW

File: records_mover/records/mover.py [hunk 1]
```diff
@@ -117,8 +117,7 @@ def move(records_source: RecordsSource,
     elif isinstance(records_source, SupportsToFileobjsSource):
         # Incompatible types in assignment (expression has type "Optional[Any]",
         #     variable has type "BaseRecordsFormat")
-        target_records_format: BaseRecordsFormat = getattr(records_target,
-                                                           "records_format", None)  # type: ignore
+        target_records_format: BaseRecordsFormat = getattr(records_target, "records_format", None)  # type: ignore
```
Comment (ryantimjohn) [inline records_mover/records/mover.py:120]: E501 line too long (114 > 100 characters)

**Comment by ryantimjohn** [comment]
@Brunope Okay, logging off but this should be good to go. Sorry to string so many together but this is based on #204

File: records_mover/records/mover.py (hunk 1 above)
Comment (Brunope) [reply records_mover/records/mover.py:120]: I agree, this line is too long

**Review by Brunope — APPROVED** [review]

File: records_mover/records/mover.py (hunk 1 above)
Comment (ryantimjohn) [reply records_mover/records/mover.py:120]: oh no, I've become the BlueLabs Butler.

**Pull request merged**